datus/agent/node/compare_node.py

```
@@ -56,24 +57,35 @@ def update_context(self, workflow: Workflow) -> Dict:
-        except Exception as e:
-            logger.error(f"SQL comparison execution error: {str(e)}")
-            return CompareResult(success=False, error=str(e), explanation="", suggest="")
+            raw_result = self.model.generate_with_json_output(messages)
+            result_dict = CompareAgenticNode._parse_comparison_output(raw_result)
+
+            return CompareResult(
+                success=True,
+                explanation=result_dict.get("explanation", "No explanation provided"),
+                suggest=result_dict.get("suggest", "No suggestions provided"),
+            )
+        except Exception as exc:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)